src/js/controller/controller.js

```diff
@@ -361,16 +361,25 @@ Object.assign(Controller.prototype, {
         function _checkPauseOnViewable(model, viewable) {
             const playerState = model.get('state');
             const adState = _getAdState();
+            const playReason = model.get('playReason');
+
+            function pauseWhenNotViewable() {
```

Comment:
Put this outside of the `_checkPauseOnViewable` function, and have `viewable` as input

Reply:
![bitmoji](https://render.bitstrips.com/v2/cpanel/3d81f2e3-dec8-4395-a284-9c21f770a7e6-b45b220b-197f-4217-b13e-41ddde95ad95-v1.png?transparent=1&palette=1&width=246)